sass/base/_structure.scss

```diff
@@ -168,7 +168,7 @@ body.app__body #root {
             }
             #sidebar-right {
                 position: relative;
-                width: 100%;
+                width: 100% !important;
```

Comment:
There's quite a few uses of `!important` here which is a bit concerning. Are they actually needed?

Reply:
In the process of refactoring, all of these were removed. :)